builder/builder.go

```diff
@@ -19,18 +19,15 @@ import (
 	common "github.com/ercole-io/ercole-agent/builder/common_builder"
 	"github.com/ercole-io/ercole-agent/config"
 	"github.com/ercole-io/ercole-agent/logger"
-	"github.com/ercole-io/ercole-agent/model"
+	"github.com/ercole-io/ercole/model"
 )
 
 // BuildData will build HostData
-func BuildData(configuration config.Configuration, version string, hostDataSchemaVersion int, log logger.Logger) *model.HostData {
+func BuildData(configuration config.Configuration, log logger.Logger) *model.HostData {
 	hostData := new(model.HostData)
 
-	hostData.Environment = configuration.Envtype
 	hostData.Location = configuration.Location
-	hostData.HostType = configuration.HostType
-	hostData.Version = version
-	hostData.HostDataSchemaVersion = hostDataSchemaVersion
```

Comment:
I moved them here: https://github.com/ercole-io/ercole-agent/blob/b72573f6dc4cc27b3216f9debb966aee5681b1cc/main.go#L68
Because this data come from main.go, and are more related to current agent running version than the builder.
Do you agree?

Reply:
Yes